README.md

```diff
@@ -56,13 +56,30 @@ Afterwards, activate the environment:
 conda activate assume-framework
 ```
 
-After these steps you can also run the example simulation:
+Quick Start
+-----------
+There are 3 ways to run an exemplar. <br>
+1.) local (without database and grafana) <br>
+2.) with docker (with database and grafana) <br>
+3.) use CLI to run simulations
```

Comment:
I would phrase that differently.
How you run the program, and which "storage" you are using for the results are two pairs of shoes ;)